src/instruments/instruments.controller.ts

```diff
@@ -34,6 +35,7 @@ import {
   filterExample,
   replaceLikeOperator,
 } from "src/common/utils";
+import { logger } from "@user-office-software/duo-logger";
```

Comment:
We should use import { Logger } from "@nestjs/common";
as the duo-logger has already been implemented in the logger module